docs/concepts/components.md

```diff
@@ -1,13 +1,70 @@
-# Lokomotive Components
+# Lokomotive component
 
-`lokoctl` can be used to manage Lokomotive *components*. A component is a k8s workload which adds
-functionality to a Lokomotive cluster. Without components, a Lokomotive cluster is just a barebone
-k8s cluster. In some cases (e.g. bare metal environments), a Lokomotive cluster may even be
-unusable without certain components.
+`lokoctl` is a command line interface for Lokomotive and its *components*. A component is a
```

Comment:
It's weird that this section has title "Lokomotive component" and it starts by talking about `lokoctl`...